modules/meta.analysis/DESCRIPTION

```diff
@@ -9,7 +9,8 @@ Authors@R: c(person("Mike","Dietze"),
              person("Dan"," Wang"),
              person("Carl", "Davidson"),
              person("Rob","Kooper"),
-             person("Shawn", "Serbin"))
+             person("Shawn", "Serbin"),
+             person("Shashank", "Singh"))
 Author: David LeBauer, Mike Dietze, Xiaohui Feng, Dan Wang, Carl
     Davidson, Rob Kooper, Shawn Serbin
 Maintainer: David LeBauer <dlebauer@illinois.edu>
```

Comment:
the `Author` and `Maintainer` sections are superseded by the `Authors@R` section -- might as well delete them while we're at it!

```suggestion
```